lib/generator.js

```diff
@@ -485,6 +490,8 @@ class Generator {
       const targetFile = path.resolve(this.targetDir, relativeSourceFile);
       const relativeTargetFile = path.relative(this.targetDir, targetFile);
 
+      if (shouldIgnoreFile(relativeSourceFile)) return;
```

Comment:
please add code comment here what we are protecting from here as I couldn't  figure out why, 
I ran all templates and was never able to get `true` for any file. I see proper protection is here https://github.com/asyncapi/generator/blob/master/lib/generator.js#L379

Reply:
This is not the only place where `generateFile` is called, that's why. I should probably remove it from the line you mentioned.